src/services/commandService.ts

```diff
@@ -564,6 +551,8 @@ export class CommandService {
     }
 
     private static createProject(isBin: boolean): void {
+        this.diagnostics.clear();
```

Comment:
There is no need for this since it is a project creation. We don't parse diagnostics here